build-logic/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt

```diff
@@ -308,6 +310,17 @@ val Project.isExperimentalTestFilteringEnabled: Boolean
     get() = systemProperty(TEST_FILTERING_ENABLED).getOrElse("false").toBoolean()
 
 
+val Project.predictiveTestSelectionEnabled: Boolean
+    get() = systemProperty(PREDICTIVE_TEST_SELECTION_ENABLED)
+        .map { it.toBoolean() }
+        .orElse(provider {
+            val protectedBranches = listOf("master", "release")
+            val branch: String? = environmentVariable("BUILD_BRANCH").orNull
```

Comment:
Can we use `project.buildBranch` here? https://github.com/gradle/gradle/blob/9b528b7ed150115df60d6c0c617e929e74de183e/build-logic/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt#L169

Reply:
I'll give it a shot.

Reply:
Done in the latest commit.